components/Common/Button/index.module.scss

```diff
@@ -0,0 +1,82 @@
+.button {
+  @apply px-[18px]
+   py-[10px]
+   font-semibold
+   relative;
+
+  &.primary {
+    @apply bg-green-600
+     text-white
+     rounded
+     border
+     border-green-600
+     shadow-sm;
+
+    &:hover {
+      @apply bg-green-700
+       border-green-700;
+    }
+
+    &:focus {
+      @apply bg-green-700
+       border-green-700;
+    }
+
+    &:disabled {
+      @apply bg-green-600
+       opacity-50;
+    }
+  }
+
+  &.secondary {
+    @apply text-neutral-800
+     rounded-lg
+     disabled:opacity-50
+     dark:text-neutral-200/100;
+
+    &:hover {
+      @apply text-neutral-800
+       bg-neutral-100
+       dark:bg-neutral-900
+       dark:text-neutral-200;
+    }
+
+    &:disabled {
+      @apply bg-neutral-100/0;
+    }
+
+    &:focus {
+      @apply text-neutral-800
+       bg-neutral-100
+       dark:bg-neutral-900
+       dark:text-neutral-200;
+    }
+  }
+
+  &.special {
+    @apply bg-green-600/10
+     text-white
+     shadow-sm
+     rounded-lg
+     border
+     border-green-600/30;
+
+    &::before {
+      @apply absolute
+       h-full
+       w-full
+       top-0
+       right-0;
+      background: radial-gradient(
```

Comment:
I wonder if using before is the right way of doing this 🤔

Also the style changes a little bit on hover and on disabled

Reply:
Is there a better way ? let me check.

Reply:
So I've looked into it, and the current layout looks great.

Can I only ask you to export this gradient into a variable?

Within `styles/new/mixins.scss` and then define a mixin for this gradient? https://sass-lang.com/documentation/at-rules/mixin/

(And the import the mixins.scss within `styles/new/index.scss`